### PR TITLE
hotter hot items

### PR DESCRIPTION
**Pull request by i-like-robots**
Use flexbox so each 'card' best fits its container without a bunch of breakpoints repeated for each combination of columns

screen shot 2015-08-06 at 11 52 03
screen shot 2015-08-06 at 11 52 56
ie8

- [x] Chrome 
- [x] Firefox
- [x] IE10
- [x] IE8-9

### CODE REVIEW

File: views/partials/special-report-item.html [hunk 1]
```diff
@@ -1,40 +1,36 @@
 <div class="hot-topics__topic hot-topics__topic--special-report" data-trackable="stream-special-report">
 	{{#ifEquals @root.featuredItems.length 1}}
 		{{#if image}}
-			<div class="hot-topics__topic__background" style="background-image: url('{{#resize 1440}}{{image.url}}{{/resize}}')"></div>
+			<!-- <div class="hot-topics__topic__background" style="background-image: url('{{#resize 1440}}{{image.url}}{{/resize}}')"></div> -->
```
Comment (ironsidevsquincy) [inline views/partials/special-report-item.html]: delete?

**Comment by ironsidevsquincy** [comment]
:+1:

**Comment by matthew-andrews** [comment]
holy moly those fonts look awful

**Comment by i-like-robots** [comment]
It's one of the lesser concerns in IE8 TBH. If we want to do core experience we could build in IE8/9 first!

**Pull request merged**